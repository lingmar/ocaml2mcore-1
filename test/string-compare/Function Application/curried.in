let x = 1 in
let foo x y = x + y in
foo x