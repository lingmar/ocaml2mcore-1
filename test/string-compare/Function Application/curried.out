mexpr
let x = 1 in
let foo = lam x. lam y.
  addi x y
in foo x
